docker/tutorial-test.sh

```diff
@@ -28,13 +28,13 @@ spack mirror add tutorial /mirror
 spack gpg trust /mirror/public.key
 spack install zlib %clang
 spack install zlib @1.2.8
-spack install zlib %gcc@4.8
+spack install zlib %gcc@7.5.0
```

Comment:
I think this should be gcc@6.5.0. There's no gcc@7.5.0 on the system when the users log in, and this isn't a compiler that we bootstrap during the tutorial.

Reply:
I picked `gcc@7.5.0`, because that's what is installed when the Dockerfile installs `gcc`, `g++` and `gfortran`.  But if I'm going to specifically install 6.5.0 too, then I can change the 4.8 in the test script to 6.5.0...

Reply:
Yeah the whole point of this one is it's demonstrating changing the compiler version, so it needs to be different from the spec of the default gcc.